gensim/models/hdpmodel.py

```diff
@@ -499,10 +502,21 @@ def hdp_to_lda(self):
 
         # beta
         beta = (self.m_lambda + self.m_eta) / (self.m_W * self.m_eta + \
-                self.m_lambda_sum[:, np.newaxis])
+                self.m_lambda_sum[:, numpy.newaxis])
 
         return (alpha, beta)
 
+
+    def suggested_lda_model(self):
+        """
+        Returns closest corresponding ldamodel object corresponding to current hdp model.
+        """
+        alpha, beta = self.hdp_to_lda()
+        ldam = ldamodel.LdaModel(num_topics=150, alpha=alpha, id2word=self.id2word)
```

Comment:
It's 150 because the default number of topics in HDP before truncation is 150. This means all the internal matrices are of that size, and to make the `expElogbeta` matrice the same shape to accept the beta, we've to initialise `ldam` to that many topics.